.gitattributes

```
@@ -1,2 +1,4 @@
 src/openapi/**/* linguist-generated=true
 openapi.json linguist-generated=true
+
+src/fonts/*.woff2 linguist-vendored=true
```

移動したときに意図不明になるので、書くならこうのがいいかも？
```suggestion
*.woff2 linguist-vendored=true
```